besu/src/test/java/org/hyperledger/besu/chainexport/RlpBlockExporterTest.java

```diff
@@ -89,7 +89,7 @@ private static Blockchain importBlocks(final BesuController controller, final Pa
 
   private static BesuController createController(final @TempDir Path dataDir) throws IOException {
     return new BesuController.Builder()
-        .fromEthNetworkConfig(EthNetworkConfig.getNetworkConfig(NetworkName.MAINNET), SyncMode.FAST)
```

Comment:
think these tests could be updated before final removal of the feature, and we could merge test changes now since that's not breaking.

Reply:
also would make the final PR smaller to review!